style.css

```diff
@@ -0,0 +1,57 @@
+html {
+    font-size: 62.5%;
+}
+
+body {
+    margin: 0 auto;
+    width: 65%;
+    text-align: center;
+    font-family: Arial, Helvetica, sans-serif;
```

Comment:
i noticed you have 2 font-family properties here. since it's a CSS file, the font-family declared last would apply, so we can safely get rid of the font-family declared first